book/SUMMARY.md

```diff
@@ -180,37 +208,3 @@
   - [EPG Ingestion](/applicaster-video-platform/content-ingestion/epg-ingestion/index.md)
   - [Collection Ingestion](/applicaster-video-platform/content-ingestion/collection-ingestion-xml/collection_via_xml.md)
   - [Authorization Providers](/applicaster-video-platform/authorization-provider/authorization_provider.md)
-
-# QuickBrick
-
-## QuickBrick
-
-- [Introduction](/quick-brick/intro.md)
-- [Apple TV](/quick-brick/tvOS/intro.md)
-  - [Plugins](/quick-brick/tvOS/Plugins/intro.md)
-    - [Analytics](/quick-brick/tvOS/Plugins/Analytics/intro.md)
-      - [Plugin creation guide](/quick-brick/tvOS/Plugins/Analytics/GeneralAnalyticsHowCreate.md)
-      - [Google Analytics](/quick-brick/tvOS/Plugins/Analytics/GoogleAnalytics/GoogleAnalytics.md)
-    - [Player](quick-brick/tvOS/Plugins/Video/intro.md)
-      - [Plugin creation guide](quick-brick/tvOS/Plugins/Video/VideoPluginsHowCreate.md)
-      - [Default Player](quick-brick/tvOS/Plugins/Video/DefaultPlayer/DefaultPlayer.md)
-      - [Player Dependent](quick-brick/tvOS/Plugins/PlayerDependents/intro.md)
-        - [Plugin creation guide](quick-brick/tvOS/Plugins/PlayerDependents/DependentPluginsHowCreate.md)
-        - [GoogleIMA](quick-brick/tvOS/Plugins/PlayerDependents/VideoAds/GoogleIMA/GoogleIMA.md)
-  - [Environment](/quick-brick/tvOS/Environment/intro.md)
-  - [Apple TV Debug](/quick-brick/tvOS/Environment/DebugBuildIntegration.md)
-  - [Apple TV Release](quick-brick/tvOS/Environment/ReleaseBuildIntegration.md)
-- [Beta Guidelines & information](/quick-brick/Beta-guidelines.md)
-- [Setup your project](/quick-brick/plugin-setup.md)
-  - [Run Quick-Brick Natively](/quick-brick/native-project-dev.md)
```

Comment:
`Run Quick-Brick Natively` should stay as well. please check where it links.

Reply:
Please check that the docs assets are still available

Reply:
> `Run Quick-Brick Natively` should stay as well. please check where it links.

Done